doc/meetings/build-wg-meeting-2019-01-29.md

```diff
@@ -0,0 +1,59 @@
+# Node.js Foundation Build WorkGroup Meeting 2019-01-29
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=ARwP_S6WhPU 
+* **GitHub Issue**: https://github.com/nodejs/build/issues/1667
+* **Minutes Google Doc**: $MINUTES_DOC$
```

Comment:
Should this be a real link or is there publishing magic that replaces it?

Reply:
actually should just remove that.